pykickstart/sections.py

```diff
@@ -254,6 +254,7 @@ def handleHeader(self, lineno, args):   # type: (PackageSection, int, List[str])
         op.add_argument("--default", dest="defaultPackages", action="store_true", default=False, introduced=F7)
         op.add_argument("--instLangs", default=None, introduced=F9)
         op.add_argument("--multilib", dest="multiLib", action="store_true", default=False, introduced=F18)
+        op.add_argument("--excludeWeakdeps", dest="excludeWeakdeps", action="store_true", default=False, introduced=F24)
```

Comment:
The F24 constant needs to be imported in the "from pykickstart.version import ..." line near the top of the file. And I assume we'd want to go with F25, since 24 is just about done.